.github/workflows/ci.yml

```diff
@@ -147,22 +146,20 @@ jobs:
           password: ${{ env.DOCKERHUB_PASSWORD }}
 
       - name: Start grafana server for e2e tests
-        if: ${{ matrix.workingDir != 'myorg-nobackend-scenesapp' }}
-        run: docker compose up -d
+        run: ANONYMOUS_AUTH_ENABLED=false docker compose up -d
         working-directory: ./${{ matrix.workingDir }}
 
       - name: Wait for grafana server
         uses: grafana/plugin-actions/wait-for-grafana@main
-        if: ${{ matrix.workingDir != 'myorg-nobackend-scenesapp' }}
+        with:
+          url: http://localhost:3000/login
```

Comment:
For my own understanding - why do we need this?

Reply:
By default, the `wait-for-grafana` Action completes when the server responds with a 200 on  `http://localhost:3000/`. However, now that anonymous auth is disabled (see [here](https://github.com/grafana/plugin-tools/blob/0f845843061c087fd1f30069d58ff1f2fd13bda6/.github/workflows/ci.yml#L149)) the server will respond with a 302, causing this Action to eventually timeout. So basically just changing the url to poll. When the login page returns a 200, the server is ready.